Source/WebCore/platform/graphics/FontCascadeCache.h

```diff
@@ -43,9 +43,9 @@ namespace WebCore {
 struct FontDescriptionKeyRareData : public RefCounted<FontDescriptionKeyRareData> {
     WTF_MAKE_FAST_ALLOCATED;
 public:
-    static Ref<FontDescriptionKeyRareData> create(FontFeatureSettings&& featureSettings, FontVariationSettings&& variationSettings, FontPalette&& fontPalette)
+    static Ref<FontDescriptionKeyRareData> create(FontFeatureSettings&& featureSettings, FontVariationSettings&& variationSettings, FontVariantAlternates&& variantAlternates, FontPalette&& fontPalette)
```

Comment:
Why are the other `font-variant-*` properties not included in this? Why is `font-variant-alternates` special? This smells fishy.

Reply:
The other ones are already included in the  `std::array<unsigned, 2> m_flags { { 0, 0 } }` (like alternates before this patch cf the change to the `makeFlagsKey` function).